matrix-client-room.el

```diff
@@ -235,7 +235,7 @@ If HTML is non-nil, treat input as HTML."
               input (concat "> <" sender "> " quoted-body "\n\n" input)
               extra-content (a-list 'format format
                                     'formatted_body formatted-body
-                                    'm.relates_to (list 'm.in_reply_to (a-list 'event_id event-id))))))
+                                    'm.relates_to (a-list 'm.in_reply_to (a-list 'event_id event-id))))))
```

Comment:
I was afraid I wasn't doing that right, because I wasn't sure of how the JSON structure mapped to the lists here.  Thanks for fixing this while I was AFK.  :)